src/app/home/home.component.html

```diff
@@ -1,11 +1,27 @@
-<app-header-static></app-header-static>
-<div class="container-top container align-center white">
-  
-  
-  <h1>
-	  EvalAI
-	</h1>
-  <div class="main-bg">
-    <img src="/assets/images/Header_bg.png" alt="">
-  </div>
+<div class="home_container"> 
```

Comment:
Don't use `_` with class names unless or until you are following CSS modular approach. And I can see that you are not using modular CSS syntax, so I prefer to use `-` instead of `_` for class names @live-wire

Reply:
Roger that 🎖